libraries/adaptive-expressions/src/functionUtils.internal.ts

```diff
@@ -84,10 +83,15 @@ export class InternalFunctionUtils {
                         if (!error) {
                             propertyName = propertyName || '';
                         }
+
+                        const sortBy = (key: string) => {
```

Comment:
It's a small optimization, but this function should be extracted out of the internal function since it isn't a closure.